views/artist-search-results.hbs

```diff
@@ -0,0 +1,10 @@
+<h4>Artist Search Result</h4>
+<div class="artist-data">
+  {{#each this}}
+    <div class="artist-search">
+      <img src="{{images}}">
```

Comment:
If you console.log {{image}}, you will see that in database you have several. images providd - it is an array. So in order to display one in the browser, you need to specify which array elemnt you are using.

 <img src="{{images.1.url}}" alt="" />